dbSearcher/dbSearcher.php

```diff
@@ -76,7 +76,7 @@ function dbs_search($search, $database, $strict = FALSE)
 					$sql_search .= implode(" OR ", $sql_search_fields);
 					$rs3 = $mysqli->query($sql_search);
 				
-					if ($rs3->num_rows > 0) {
+					if ($rs3 && $rs3->num_rows > 0) {
```

Comment:
This is a hackfix. 

Underlying issue lies in this scope:

```
					if ($rs2->num_rows > 0) {
						while ($r2 = $rs2->fetch_array()){
							$column = $r2[0];

							if ($strict)
								$sql_search_fields[] = $column." = ".$search;
							else 
								$sql_search_fields[] = $column." LIKE('%".$search."%')";
						}
						
						$rs2->close();
					}
```

Reply:
I think this is outside the scope of the current PR. The problem you mentioned in https://github.com/PatricNox/DatabaseSearcher/pull/14#issuecomment-542860139 is a more generic problem on variable escaping.

I'm preparing a new PR which rewrites the `dbs_search` function and adds quotes in variables and some more descriptive variable names, if you're interested.